data/scripting/win_conditions/wood_gnome.lua

```diff
@@ -19,7 +19,10 @@ local wc_desc = _(
 [[As wood gnome you like big forests, so your task is to have more trees on ]] ..
 [[your territory than any other player. The game will end after 4 hours of ]] ..
 [[playing. The one with the most trees at that point will win the game.]])
-local wc_trees_owned = _"Trees owned"
+local wc_trees_owned = "Trees owned"
+-- This needs to be exactly like wc_trees_owned, but localized, because wc_trees_owned
+-- will be used as the key to fetch the translation in C++
+local wc_trees_owned_i18n = _"Trees owned"
 
 local r = {
    name = wc_name,
```

Comment:
```suggestion
   name = wc_trees_owned,
```

I haven't tested this for all win conditions, looks like we need more changes here

Reply:
This PR focuses on the statistics button mentioned in the issue. Other parts of the win conditions will still be stored translated in the save. This includes the task or win message, which are a bit trickier to translate afterwards. I don't know what your suggestion should change, but I checked the statistics button with all win conditions and it translated fine.

Reply:
Thanks for confirming it's tested, let's have this branch :)